modules/ServiceCell.js

```diff
@@ -15,7 +15,11 @@
    * @class  ServiceCell
    * @author kent.willis@vecna.com
    */
-  var ServiceCell = Cell.extend({ });
+  var ServiceCell = Cell.extend({
+    __postInitialize: function() {
+      registry.serviceInitialized(this);
```

Comment:
registry is not imported (found while testing this... definitely needs unit tests).